XPlat/Google.Grpc/build.cake

```diff
@@ -0,0 +1,205 @@
+	},
+};
+string ARTIFACT_FILE = "";
+string NUGET_VERSION=$"{ARTIFACT_VERSION}";
+
+
+BuildSpec buildSpec = new BuildSpec () 
+{
+	Libs = new ISolutionBuilder [] 
+	{
+		new DefaultSolutionBuilder 
+		{
+			SolutionPath = "./source/Xamarin.Grpc.sln",
+			OutputFiles = new [] 
+			{ 
+				new OutputFileCopy 
+				{ 
+					FromFile = "./source/Xamarin.Grpc.Core.Bindings.XamarinAndroid/bin/Release/monoandroid81/Xamarin.Grpc.Core.dll" 
+				},
+				new OutputFileCopy 
+				{ 
+					FromFile = $"source/Xamarin.Grpc.Core.Bindings.XamarinAndroid/bin/Release/Xamarin.Grpc.Core.{ARTIFACT_VERSION}.nupkg" 
+				},
+				new OutputFileCopy 
+				{ 
+					FromFile = "./source/Xamarin.Grpc.Stub.Bindings.XamarinAndroid/bin/Release/monoandroid81/Xamarin.Grpc.Stub.dll" 
+				},
+				new OutputFileCopy 
+				{ 
+					FromFile = $"source/Xamarin.Grpc.Stub.Bindings.XamarinAndroid/bin/Release/Xamarin.Grpc.Stub.{ARTIFACT_VERSION}.nupkg" 
+				},
+
+				new OutputFileCopy 
+				{ 
+					FromFile = "./source/Xamarin.Grpc.Protobuf.Lite.Bindings.XamarinAndroid/bin/Release/monoandroid81/Xamarin.Grpc.Protobuf.Lite.dll" 
+				},
+				new OutputFileCopy 
+				{ 
+					FromFile = $"source/Xamarin.Grpc.Protobuf.Lite.Bindings.XamarinAndroid/bin/Release/Xamarin.Grpc.Protobuf.Lite.{ARTIFACT_VERSION}.nupkg" 
+				},
+				new OutputFileCopy 
+				{ 
+					FromFile = "./source/Xamarin.Grpc.OkHttp.Bindings.XamarinAndroid/bin/Release/monoandroid81/Xamarin.Grpc.OkHttp.dll" 
+				},
+				new OutputFileCopy 
+				{ 
+					FromFile = $"source/Xamarin.Grpc.OkHttp.Bindings.XamarinAndroid/bin/Release/Xamarin.Grpc.OkHttp.{ARTIFACT_VERSION}.nupkg" 
+				},
+
+
+
+			}
+		}
+	},
+
+	Samples = new ISolutionBuilder [] 
+	{
+		new DefaultSolutionBuilder 
+		{ 
+			SolutionPath = "./samples/Xamarin.Grpc.sln" 
+		},	
+	},
+};
+
+Task ("externals")
+	.IsDependentOn ("externals-base")
+	.Does 
+	(
+		() => 
+		{
+			Information("externals ...");
+			EnsureDirectoryExists("./externals/android");
+			
+			foreach(KeyValuePair<string, string> JAR_URL_ARTIFACT_FILE in JAR_URLS_ARTIFACT_FILES)
+			{
+				string JAR_URL = JAR_URL_ARTIFACT_FILE.Key;
+				string ARTIFACT_FILE = JAR_URL_ARTIFACT_FILE.Value;
+
+				Information($"    downloading ...");
+				Information($"                {JAR_URL}");
+				Information($"    to ");
+				Information($"                {ARTIFACT_FILE}");
+				if ( ! string.IsNullOrEmpty(JAR_URL) && ! FileExists(ARTIFACT_FILE))
+				{
+					DownloadFile (JAR_URL, ARTIFACT_FILE);
+				}
+
+			}
+		}
+	);
+
+
+Task ("clean")
+	.IsDependentOn ("clean-base")
+	.Does
+	(
+		() => 
+		{
+			if (DirectoryExists ("./externals/"))
+			{
+				DeleteDirectory ("./externals", true);
+			}
+		}
+	);
+
+Task("nuget")
+	.IsDependentOn("libs")
+	.Does
+	(
+		() =>
+		{
+			EnsureDirectoryExists("./output");
+
+			MSBuild
+			(
+				"./source/Xamarin.Protobuf.Lite.Bindings.XamarinAndroid/Xamarin.Protobuf.Lite.Bindings.XamarinAndroid.csproj", 
```

Comment:
This path is not the correct path for the project

./source/Xamarin.Grpc.Protobuf.Lite.Bindings.XamarinAndroid/Xamarin.Grpc.Protobuf.Lite.Bindings.XamarinAndroid.csproj